lib/muffin_man/catalog_items/v20201201.rb

```diff
@@ -17,7 +17,7 @@ class V20201201 < SpApiClient
       ].freeze
       GET_CATALOG_ITEM_PARAMS = %w[includedData locale].freeze
 
-      def search_catalog_items(keywords, marketplace_ids, params = {})
+      def search_catalog_items(keywords, marketplace_ids, params = {}, version="2020-12-01")
```

Comment:
Again, don't make the version passed in to the already versioned class name. We want to keep the gems naming and structure consistent. If a base class is needed, implement that.